style.css

```diff
@@ -58,7 +58,42 @@ ul {
   margin-right: 10px;
 }
 
-nav ul {
+.mobile-menu {
+  background: #6070ff;
+  position: absolute;
```

Comment:
- Hey, kindly make this `fixed` so that the height can take up the whole height of the device to improve your design.

Reply:
kindly tell me do you want me to fix the position @zieeco

Reply:
@Afsheen96, change the position from absolute to fixed.

Reply:
i did it thanks